miRge.pl

```diff
@@ -36,7 +36,7 @@
 my $version = "2.0";
 local $ENV{PATH} = "$ENV{PATH}:".$miRgePath;
 $miRgePath =~ s/\/[^\/]+\.pl/\//;
-my $refPath = $miRgePath."miRge.seqLibs/";
+my $refPath = $miRgePath."usr/local/miRge/miRge.seqLibs/";
```

Comment:
Thanks Alex. I just downloaded the new .tar.gz and am still getting usr/local/miRge/miRge.seqLibs/.